packages/dds/tree/src/simple-tree/api/schemaFactory.ts

```diff
@@ -362,6 +364,12 @@ export class SchemaFactory<
 		T
 	>;
 
+	/**
+	 * @privateRemarks
+	 * This should return TreeNodeSchemaBoth, however TypeScript gives an error if one of the overloads implicitly up casts the return type of the implementation.
+	 * This seems like a TypeScript bug getting variance backwards for overload return types since its erroring when the relation between the overload
```

Comment:
Nit ```suggestion
	 * This seems like a TypeScript bug getting variance backwards for overload return types since it's erroring when the relation between the overload
```